src/oaklib/interfaces/differ_interface.py

```diff
@@ -282,3 +305,345 @@ def compare_term_in_two_ontologies(
         self, other_ontology: BasicOntologyInterface, curie: CURIE, other_curie: CURIE = None
     ) -> Any:
         raise NotImplementedError
+
+    def diff_structured(
+        self,
+        other_ontology: BasicOntologyInterface,
+        configuration: DiffConfiguration = None,
+    ) -> Iterator[Dict[str, Any]]:
+        """
+        Provides a structured diff of two ontologies
+
+        Required sequence of changes:
```

Comment:
"required" sounds like it's a required input

Reply:
Switched to "Preferred"